server/src/main/java/com/cloud/template/TemplateManagerImpl.java

```diff
@@ -1302,19 +1302,21 @@ private boolean attachISOToVM(long vmId, long isoId, boolean attach) {
 
         Command cmd = null;
         if (attach) {
-            cmd = new AttachCommand(disk, vmName, vmTO.getDetails());
```

Comment:
Hi all, 
I'm sorry to ask on already merged PR but is the `getDetails()` breaks the force functionality on VMware? It's used on KVM when the VM boot type is UEFI, and now it causes problems when detaching/attaching the ISO.

Reply:
@slavkap I think this got reverted while rebasing the code. Thanks for pointing it out. This would need to be fixed.

Reply:
Thanks, @Pearl1594, for the answer! I could push a fix for this, but it has to be on 4.15.2 (when it's open) or on 4.16?